File: kotlin-spring/src/main/kotlin/org/openapitools/model/InvalidMessageError.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.AnyType
import org.openapitools.model.InvalidMessageErrorAllOf
import org.openapitools.model.JsonSuccessBase
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param result 
 * @param msg 
 * @param rawContent The raw content of the message. 
 */
data class InvalidMessageError(

    @field:Valid
    @field:JsonProperty("result", required = true) val result: AnyType,

    @field:Valid
    @field:JsonProperty("msg", required = true) val msg: AnyType,

    @field:JsonProperty("raw_content") val rawContent: kotlin.String? = null
) {

}

File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/InvalidMessageError.scala
package org.openapitools.server.model


/**
 * @param result  for example: ''null''
 * @param msg  for example: ''null''
 * @param rawContent The raw content of the message.  for example: ''null''
*/
final case class InvalidMessageError (
  result: AnyType,
  msg: AnyType,
  rawContent: Option[String]
)

